### PR TITLE
adding new api for cert retrieval to include hp use case

### PR DESCRIPTION
**Pull request by saiprakash-v**
@egov-joy @samarthgr

### CODE REVIEW

File: default-configuration/etcd/HealthProfessionalCertificate/programParams.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "groupBy": ""
```
Comment (egov-joy) [inline default-configuration/etcd/HealthProfessionalCertificate/programParams.json:2]: do we need an empty value if there is no grouping required or can this configuration be omitted from etcd?

File: backend/certificate_api/src/services/configuration_service.js [hunk 2]
```diff
@@ -2,12 +2,14 @@ const {Etcd3} = require('etcd3');
 const sanitizeHtml = require('sanitize-html');
 const Handlebars = require('handlebars');
 const config = require('../../configs/config');
-const {TEMPLATES, EU_VACCINE_CONFIG_KEYS,HELPERS} = require('../../configs/constants');
+const {TEMPLATES, EU_VACCINE_CONFIG_KEYS,HELPERS, TEMPLATE_KEY, PARAMS_KEY, HELPER_FUNCTIONS_KEY} = require('../../configs/constants');
 let etcdClient;
 let configuration;
 let vaccineCertificateTemplate = null, testCertificateTemplate = null, euVaccineCertificateTemplate = null;
+let healthProfessionalCertificateTemplate = null;
```
Comment (egov-joy) [inline backend/certificate_api/src/services/configuration_service.js:9]: is this required any more?

File: default-configuration/etcd/HealthProfessionalCertificate/programParams.json (hunk 1 above)
Comment (saiprakash-v) [reply default-configuration/etcd/HealthProfessionalCertificate/programParams.json:2]: we can remove it

**Review by egov-joy — APPROVED** [review]

File: backend/certificate_api/src/services/configuration_service.js (hunk 2 above)
Comment (saiprakash-v) [reply backend/certificate_api/src/services/configuration_service.js:9]: no. forgot to remove this. Just pushed a new commit removing this and groupBy

**Review by samarthgr — APPROVED** [review]

**Pull request merged**